plugins/UFM_NDT_Plugin/ufm_sim_web_service/topo_diff/topo_diff.py

```diff
@@ -283,6 +291,58 @@ def get_request(host_ip, protocol, resource, headers):
     logging.info("UFM API Request Status: {}, URL: {}".format(response.status_code, request))
     return response
 
+
+def save_cable_validation_cookies(requests_cookiejar, cookie_filename):
+    '''
+    Save cookies to file
+    :param requests_cookiejar:
+    :param cookie_filename:
+    '''
+    with open(cookie_filename, 'wb') as f:
+        pickle.dump(requests_cookiejar, f)
+
+def load_cable_validation_cookies(cookie_filename):
+    '''
+    Load cookie from file
+    :param cookie_filename:
+    '''
+    with open(cookie_filename, 'rb') as f:
+        return pickle.load(f)
+
+def post_request_with_cookies(host_addr, port_num, username, password):
+    '''
+    Create cookie file and then use it for get request
+    :param host_addr:
+    :param port_num:
+    :param username:
+    :param password:
+    :param url:
+    '''
+    send_url = f"{CABLE_VALIDATION_PROTOCOL}://{host_addr}:{port_num}/{CABLE_VALIDATION_LOGIN_URL}"
+    send_payload = {'httpd_username': username, 'httpd_password': password}
+    try:
+        session = requests.Session()
```

Comment:
add log message when sending the request to the cable validation.

Reply:
added